pkg/payload/payload.go

```diff
@@ -257,6 +259,7 @@ func GetImplicitlyEnabledCapabilities(updatePayloadManifests []manifest.Manifest
 					}
 				}
 			}
+			break
```

Comment:
I don't understand this `break` or the one a few lines up^^.  Can you explain why we want them in the commit message?

Reply:
Removed breaks such that all update manifests are compared against all current manifests, i.e. handles duplicate manifest resources.